package.json

```diff
@@ -61,6 +60,7 @@
     "webpack-node-externals": "^1.7.2"
   },
   "dependencies": {
+    "@types/node": "^13.1.1",
```

Comment:
Was this intentional? Because is in the dependencies whoever install ts-auto-mock will also install types/node.